doc/configuration.rst

```diff
@@ -465,10 +465,59 @@ seconds, the following can be used:
 
     confluence_timeout = 10
 
+advanced configuration - formatting
```

Comment:
Will take a look a the storage format - I was not aware of that.  Removed the formatting from this PR.

Reply:
After a good night's sleep I realised that I was talking poppy-cock.  The ordering of 'input' is fixed because it is defined by the storage format so, for example, the `http://www.atlassian.com` and the `Atlassian` are always in the same order and the plugin code knows that order.  So a simple array will always work as the plugin will know to...
* Put in formatting [0] (`<a href="`)
* now the `http://www.atlassian.com` input
* now formatting [1] (`">`)
* now the `Atlassian` input
* and end with formatting [2] (`</a>`)
et voila!